### DISCUSSION TITLE
Would libigl support Apple Silicon (M1 chip)?

### DISCUSSION THREAD

**Discussion by Xayah-Hina** [libigl/libigl#1908] [General]
Would libigl support Apple Silicon (M1 chip)? : )

**Comment by alecjacobson** [comment]
I've been exploring this. Seems like most things will work fine. There are
a couple functions relying on SSE intrinsics (polar_svd and FWN). polar_svd
has a non-vectorized fallback, but FWN does not (yet).

Best to use it in header only mode (i.e., not as a static library), so that
you can enjoy just the parts that work.

-Alec

>
